UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -667,32 +680,38 @@ public void DODMG()
 		}
 
 		public float NutrimentConsumed = 0;
+		[SerializeField] private bool stopOverallCalculation = false;
+		[SerializeField] private bool stopMetabolism = false;
 
 		//Server Side only
 		private void PeriodicUpdate()
 		{
 			NutrimentConsumed = 0;
 			for (int i = BodyPartList.Count - 1; i >= 0; i--)
 			{
-				BodyPartList[i].ImplantPeriodicUpdate();
+				if (!stopMetabolism)
+					BodyPartList[i].ImplantPeriodicUpdate();
 			}
 
 
 			foreach (var system in ActiveSystems)
```

Comment:
wait, wouldn't this stop every other system? This doesn't target only metabolism, it targets all health systems.

Reply:
Yes? Will be better rename that to something other